docs/sources/flow/reference/components/beyla.ebpf.md

```diff
@@ -0,0 +1,250 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/reference/components/beyla.ebpf/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/beyla.ebpf/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/reference/components/beyla.ebpf/
+- /docs/grafana-cloud/send-data/agent/flow/reference/components/beyla.ebpf/
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/beyla.ebpf/
+description: Learn about beyla.ebpf
+title: beyla.ebpf
+---
+
+# beyla.ebpf
+
+The `beyla.ebpf` component is used a wrapper of [Grafana Beyla][] which collects metrics and traces from eBPF programs running on the host. The component can be configured to collect data from a specific port or executable name, and other criteria from Kubernetes metadata. The component can  be scraped by Prometheus and can send traces to an OpenTelemetry Collector.
```

Comment:
Do we want to explicitly mention that OTel metrics are not supported at the moment? It might be easy to overlook...

Reply:
+1 this would be good to mention, as it's not immediately clear to me. 

I think this is best mentioned in the section for the `output` block.

Reply:
Clarified in the `output` section. @grcevski let me know if it's clear enough